cli/internal/test/cli.go

```diff
@@ -42,7 +44,12 @@ func (c *FakeCli) Out() *command.OutStream {
 	return command.NewOutStream(c.out)
 }
 
-// In returns thi input stream the cli will use
+// In returns the input stream the cli will use
 func (c *FakeCli) In() *command.InStream {
 	return command.NewInStream(c.in)
 }
+
+// ConfigFile returns the config file the cli will use
+func (c *FakeCli) ConfigFile() *configfile.ConfigFile {
+	return config.NewConfigFile("configFilename")
```

Comment:
I choose another path for that in https://github.com/docker/docker/compare/master...vdemeester:volume-unit-tests#diff-ecd4d24831d477dbaef6c122698aa6ecR69 👼